interp/modintern.mli

```diff
@@ -33,5 +34,8 @@ exception ModuleInternalizationError of module_internalization_error
 
 type module_kind = Module | ModType | ModAny
 
+val intern_module_ast :
```

Comment:
In general I'd be nice if you could document the newly added functions

Reply:
Sure, I added a comment.